src/AMSlib/wf/debug.h

```diff
@@ -25,8 +25,11 @@ void memUsage(double& vm_usage, double& resident_set);
               "[AMS:%s:%s] ",                      \
               ams::util::getVerbosityKey(vl),      \
               GETNAME(id));                        \
+      fflush(ams::util::out(vl));                  \
```

Comment:
is `fflush` necessary? I am guessing this is because multi-thread execution. But it may be too high of a penalty. Let me know which use case you had in mind when you started flushing?

Reply:
Yes I wanted to fix the printing which can be messed up and I tried a few things but it's more complicated than just flush so I gave up. But I forgot to remove the fflush. Good catch.